2018-07-02 23:57:39.558 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:57:39.561 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-02 23:57:39.608 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:57:39.609 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-02 23:57:39.611 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:57:40.668 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.058s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:57:40.671 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:57:41.692 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.021s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:57:41.694 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:57:42.808 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.114s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:57:42.811 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:57:43.800 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.989s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:58:39.563 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:58:39.565 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-02 23:58:39.620 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:58:39.620 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-02 23:58:39.622 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:58:40.736 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.115s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:58:40.739 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:58:41.841 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.102s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:58:41.844 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:58:43.120 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.277s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:58:43.123 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:58:44.271 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.148s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:59:39.565 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:59:39.567 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-02 23:59:39.600 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-02 23:59:39.601 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-02 23:59:39.602 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:59:40.790 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.188s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:59:40.792 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:59:41.809 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.017s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:59:41.813 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:59:42.873 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.061s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-02 23:59:42.877 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-02 23:59:43.992 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.116s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:00:39.561 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:00:39.565 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:00:39.612 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:00:39.614 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:00:39.615 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:00:40.872 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.257s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:00:40.874 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:00:42.037 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.163s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:00:42.040 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:00:43.135 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.095s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:00:43.140 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:00:44.320 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.181s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:01:39.564 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:01:39.566 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:01:39.611 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:01:39.612 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:01:39.614 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:01:40.818 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.205s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:01:40.820 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:01:41.883 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.063s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:01:41.888 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:01:42.964 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.077s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:01:42.968 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:01:43.986 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.019s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:21.408 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Volume reschedule parameters: True retry: {u'num_attempts': 1, u'backends': [u'localhost.localdomain@lvm#lvm'], u'hosts': [u'localhost.localdomain@lvm#lvm']} get_flow /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:1075
2018-07-03 00:02:21.498 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Flow 'volume_create_manager' (e20493f8-d580-49d8-a5f4-217f5259d1f6) transitioned into state 'RUNNING' from state 'PENDING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-07-03 00:02:21.513 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (96efc9f1-aa50-429b-bb2f-ac4a72b6336b) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:21.590 803 DEBUG oslo_db.sqlalchemy.engines [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-07-03 00:02:21.671 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (96efc9f1-aa50-429b-bb2f-ac4a72b6336b) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'Volume(_name_id=None,admin_metadata={},attach_status='detached',availability_zone='nova',bootable=False,cluster=<?>,cluster_name=None,consistencygroup=<?>,consistencygroup_id=None,created_at=2018-07-02T22:02:20Z,deleted=False,deleted_at=None,display_description=None,display_name='tempest-VOLUME_SAMPLE-1530568629',ec2_id=None,encryption_key_id=None,glance_metadata=<?>,group=<?>,group_id=None,host='localhost.localdomain@lvm#lvm',id=9418bd73-30a7-4927-b450-e2af3cfb105d,launched_at=None,metadata={},migration_status=None,multiattach=False,previous_status=None,project_id='fc87b8481ee14800a696810d32b79c6f',provider_auth=None,provider_geometry=None,provider_id=None,provider_location=None,replication_driver_data=None,replication_extended_status=None,replication_status=None,scheduled_at=2018-07-02T22:02:21Z,size=1,snapshot_id=None,snapshots=<?>,source_volid=None,status='creating',terminated_at=None,updated_at=2018-07-02T22:02:21Z,user_id='cd8907a7c13344eb864b011f50896a6e',volume_attachment=<?>,volume_type=VolumeType(4498be6b-fc65-4290-afaf-2e418f07d120),volume_type_id=4498be6b-fc65-4290-afaf-2e418f07d120)' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:21.695 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (e17fc9db-951a-46ff-afa6-98d2eb5d938e) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:21.710 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (e17fc9db-951a-46ff-afa6-98d2eb5d938e) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:21.715 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (f4ae6980-d66d-4cd1-baa5-c30bd3f13c52) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:21.744 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:22.911 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:22.937 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:22 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1530568629", "tags": [], "container_format": "bare", "created_at": "2018-07-02T21:57:53Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-07-02T21:57:54Z", "visibility": "public", "self": "/v2/images/4b995790-8d09-478f-9121-60efe5389473", "min_disk": 0, "protected": false, "id": "4b995790-8d09-478f-9121-60efe5389473", "file": "/v2/images/4b995790-8d09-478f-9121-60efe5389473/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "007e7f1ed2554995b505b934dd328646", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:22.939 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:23.002 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/schemas/image used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:23.004 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4350
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:22 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Stato dell'immagine"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "Elenco di stringhe relative all'immagine"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Formato del contenitore"}, "min_ram": {"type": "integer", "description": "Quantit\u00e0 di ram (in MB) richiesta per l'immagine di avvio."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "Un insieme di URL per accedere al file di immagini conservato nell'archivio esterno"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Ambito di accessibilit\u00e0 dell'immagine"}, "updated_at": {"readOnly": true, "type": "string", "description": "Data e ora dell'ultima modifica dell'immagine"}, "owner": {"type": ["null", "string"], "description": "Proprietario dell'immagine", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "Un URL al file di immagini"}, "min_disk": {"type": "integer", "description": "Quantit\u00e0 di spazio su disco (in GB) richiesto per l'immagine di avvio."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione virtuale dell'immagine in byte"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "Un identificativo per l'immagine"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione del file di immagine in byte"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Nome descrittivo per l'immagine", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "hash md5 del contenuto dell'immagine. ", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Data e ora della registrazione dell'immagine"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Formato del disco"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "Se true, l'immagine non sar\u00e0 eliminabile."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL per accedere al file di immagini tenuto nell'archivio esterno"}, "self": {"readOnly": true, "type": "string", "description": "Un URL personale all'immagine"}, "schema": {"readOnly": true, "type": "string", "description": "Un URL allo schema di immagini"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:23.024 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:23.176 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:23.178 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:23 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1530568629", "tags": [], "container_format": "bare", "created_at": "2018-07-02T21:57:53Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-07-02T21:57:54Z", "visibility": "public", "self": "/v2/images/4b995790-8d09-478f-9121-60efe5389473", "min_disk": 0, "protected": false, "id": "4b995790-8d09-478f-9121-60efe5389473", "file": "/v2/images/4b995790-8d09-478f-9121-60efe5389473/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "007e7f1ed2554995b505b934dd328646", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:23.179 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:23.201 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/schemas/image used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:23.211 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4350
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:23 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Stato dell'immagine"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "Elenco di stringhe relative all'immagine"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Formato del contenitore"}, "min_ram": {"type": "integer", "description": "Quantit\u00e0 di ram (in MB) richiesta per l'immagine di avvio."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "Un insieme di URL per accedere al file di immagini conservato nell'archivio esterno"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Ambito di accessibilit\u00e0 dell'immagine"}, "updated_at": {"readOnly": true, "type": "string", "description": "Data e ora dell'ultima modifica dell'immagine"}, "owner": {"type": ["null", "string"], "description": "Proprietario dell'immagine", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "Un URL al file di immagini"}, "min_disk": {"type": "integer", "description": "Quantit\u00e0 di spazio su disco (in GB) richiesto per l'immagine di avvio."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione virtuale dell'immagine in byte"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "Un identificativo per l'immagine"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione del file di immagine in byte"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Nome descrittivo per l'immagine", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "hash md5 del contenuto dell'immagine. ", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Data e ora della registrazione dell'immagine"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Formato del disco"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "Se true, l'immagine non sar\u00e0 eliminabile."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL per accedere al file di immagini tenuto nell'archivio esterno"}, "self": {"readOnly": true, "type": "string", "description": "Un URL personale all'immagine"}, "schema": {"readOnly": true, "type": "string", "description": "Un URL allo schema di immagini"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:23.230 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:23.259 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/schemas/image used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:23.261 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4350
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:23 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Stato dell'immagine"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "Elenco di stringhe relative all'immagine"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Formato del contenitore"}, "min_ram": {"type": "integer", "description": "Quantit\u00e0 di ram (in MB) richiesta per l'immagine di avvio."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "Un insieme di URL per accedere al file di immagini conservato nell'archivio esterno"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Ambito di accessibilit\u00e0 dell'immagine"}, "updated_at": {"readOnly": true, "type": "string", "description": "Data e ora dell'ultima modifica dell'immagine"}, "owner": {"type": ["null", "string"], "description": "Proprietario dell'immagine", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "Un URL al file di immagini"}, "min_disk": {"type": "integer", "description": "Quantit\u00e0 di spazio su disco (in GB) richiesto per l'immagine di avvio."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione virtuale dell'immagine in byte"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "Un identificativo per l'immagine"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione del file di immagine in byte"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Nome descrittivo per l'immagine", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "hash md5 del contenuto dell'immagine. ", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Data e ora della registrazione dell'immagine"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Formato del disco"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "Se true, l'immagine non sar\u00e0 eliminabile."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL per accedere al file di immagini tenuto nell'archivio esterno"}, "self": {"readOnly": true, "type": "string", "description": "Un URL personale all'immagine"}, "schema": {"readOnly": true, "type": "string", "description": "Un URL allo schema di immagini"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:23.265 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (f4ae6980-d66d-4cd1-baa5-c30bd3f13c52) transitioned into state 'SUCCESS' from state 'RUNNING' with result '{'status': u'creating', 'image_location': (None, None), 'image_id': u'4b995790-8d09-478f-9121-60efe5389473', 'volume_id': u'9418bd73-30a7-4927-b450-e2af3cfb105d', 'volume_size': 1, 'image_service': <cinder.image.glance.GlanceImageService object at 0x7f089099e650>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1530568629', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 7, 2, 21, 57, 53, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 7, 2, 21, 57, 54, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'007e7f1ed2554995b505b934dd328646', u'protected': False, u'id': u'4b995790-8d09-478f-9121-60efe5389473', u'file': u'/v2/images/4b995790-8d09-478f-9121-60efe5389473/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}, 'volume_name': u'volume-9418bd73-30a7-4927-b450-e2af3cfb105d', 'type': 'image'}' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:23.269 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (cc51f978-8e0a-4b0a-a2f8-8fc3ff1377e6) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:23.433 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (cc51f978-8e0a-4b0a-a2f8-8fc3ff1377e6) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:23.437 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (df8cb4e7-6686-410d-af03-a146e8e31dff) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:23.438 803 INFO cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Volume 9418bd73-30a7-4927-b450-e2af3cfb105d: being created as image with specification: {'status': u'creating', 'image_location': (None, None), 'volume_size': 1, 'volume_name': u'volume-9418bd73-30a7-4927-b450-e2af3cfb105d', 'image_id': u'4b995790-8d09-478f-9121-60efe5389473', 'image_service': <cinder.image.glance.GlanceImageService object at 0x7f089099e650>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1530568629', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 7, 2, 21, 57, 53, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 7, 2, 21, 57, 54, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'007e7f1ed2554995b505b934dd328646', u'protected': False, u'id': u'4b995790-8d09-478f-9121-60efe5389473', u'file': u'/v2/images/4b995790-8d09-478f-9121-60efe5389473/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}}
2018-07-03 00:02:23.439 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Cloning 9418bd73-30a7-4927-b450-e2af3cfb105d from image 4b995790-8d09-478f-9121-60efe5389473  at location (None, None). _create_from_image /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:860
2018-07-03 00:02:23.442 803 DEBUG cinder.coordination [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "/var/lib/cinder/cinder-4b995790-8d09-478f-9121-60efe5389473" acquired by "_create_from_image_cache_or_download" :: waited 0.002s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-07-03 00:02:23.445 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473/file log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:23.521 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473/file used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:23.523 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/octet-stream
Content-Md5: 443b7623e27ecf03dc9e01ee93f67afe
Content-Length: 12716032
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:23 GMT
Connection: keep-alive
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:23.960 803 DEBUG cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Image fetch details: dest /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm, size 12.00 MB, duration 1.00 sec fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:247
2018-07-03 00:02:23.962 803 INFO cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Image download 12.00 MB at 12.00 MB/s
2018-07-03 00:02:23.966 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:24.072 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:24.074 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:24 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1530568629", "tags": [], "container_format": "bare", "created_at": "2018-07-02T21:57:53Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-07-02T21:57:54Z", "visibility": "public", "self": "/v2/images/4b995790-8d09-478f-9121-60efe5389473", "min_disk": 0, "protected": false, "id": "4b995790-8d09-478f-9121-60efe5389473", "file": "/v2/images/4b995790-8d09-478f-9121-60efe5389473/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "007e7f1ed2554995b505b934dd328646", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:24.075 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:24.095 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/schemas/image used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:24.097 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4350
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:24 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Stato dell'immagine"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "Elenco di stringhe relative all'immagine"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Formato del contenitore"}, "min_ram": {"type": "integer", "description": "Quantit\u00e0 di ram (in MB) richiesta per l'immagine di avvio."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "Un insieme di URL per accedere al file di immagini conservato nell'archivio esterno"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Ambito di accessibilit\u00e0 dell'immagine"}, "updated_at": {"readOnly": true, "type": "string", "description": "Data e ora dell'ultima modifica dell'immagine"}, "owner": {"type": ["null", "string"], "description": "Proprietario dell'immagine", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "Un URL al file di immagini"}, "min_disk": {"type": "integer", "description": "Quantit\u00e0 di spazio su disco (in GB) richiesto per l'immagine di avvio."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione virtuale dell'immagine in byte"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "Un identificativo per l'immagine"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione del file di immagine in byte"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Nome descrittivo per l'immagine", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "hash md5 del contenuto dell'immagine. ", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Data e ora della registrazione dell'immagine"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Formato del disco"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "Se true, l'immagine non sar\u00e0 eliminabile."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL per accedere al file di immagini tenuto nell'archivio esterno"}, "self": {"readOnly": true, "type": "string", "description": "Un URL personale all'immagine"}, "schema": {"readOnly": true, "type": "string", "description": "Un URL allo schema di immagini"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:24.102 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:25.440 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm" returned: 0 in 1.339s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:25.443 803 DEBUG cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Temporary image 4b995790-8d09-478f-9121-60efe5389473 is fetched for user cd8907a7c13344eb864b011f50896a6e. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:676
2018-07-03 00:02:25.444 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:26.642 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm" returned: 0 in 1.198s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:26.646 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-9418bd73-30a7-4927-b450-e2af3cfb105d cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:28.016 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-9418bd73-30a7-4927-b450-e2af3cfb105d cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.370s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:28.071 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Attempting download of 4b995790-8d09-478f-9121-60efe5389473 ((None, None)) to volume 9418bd73-30a7-4927-b450-e2af3cfb105d. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:536
2018-07-03 00:02:28.076 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:28.133 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/images/4b995790-8d09-478f-9121-60efe5389473 used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:28.136 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:28 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1530568629", "tags": [], "container_format": "bare", "created_at": "2018-07-02T21:57:53Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-07-02T21:57:54Z", "visibility": "public", "self": "/v2/images/4b995790-8d09-478f-9121-60efe5389473", "min_disk": 0, "protected": false, "id": "4b995790-8d09-478f-9121-60efe5389473", "file": "/v2/images/4b995790-8d09-478f-9121-60efe5389473/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "007e7f1ed2554995b505b934dd328646", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:28.138 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}f26b44ce74fd15ca7c8c554b20741fe2ea24ca91' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800' http://10.0.20.23:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-07-03 00:02:28.161 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] GET call to image for http://10.0.20.23:9292/v2/schemas/image used request id req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-07-03 00:02:28.165 803 DEBUG glanceclient.common.http [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4350
X-Openstack-Request-Id: req-cd56eb91-6afd-4a75-a0eb-53e83fba5800
Date: Mon, 02 Jul 2018 22:02:28 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Stato dell'immagine"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "Elenco di stringhe relative all'immagine"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Formato del contenitore"}, "min_ram": {"type": "integer", "description": "Quantit\u00e0 di ram (in MB) richiesta per l'immagine di avvio."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "Un insieme di URL per accedere al file di immagini conservato nell'archivio esterno"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Ambito di accessibilit\u00e0 dell'immagine"}, "updated_at": {"readOnly": true, "type": "string", "description": "Data e ora dell'ultima modifica dell'immagine"}, "owner": {"type": ["null", "string"], "description": "Proprietario dell'immagine", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "Un URL al file di immagini"}, "min_disk": {"type": "integer", "description": "Quantit\u00e0 di spazio su disco (in GB) richiesto per l'immagine di avvio."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione virtuale dell'immagine in byte"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "Un identificativo per l'immagine"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Dimensione del file di immagine in byte"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Nome descrittivo per l'immagine", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "hash md5 del contenuto dell'immagine. ", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Data e ora della registrazione dell'immagine"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Formato del disco"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "Se true, l'immagine non sar\u00e0 eliminabile."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL per accedere al file di immagini tenuto nell'archivio esterno"}, "self": {"readOnly": true, "type": "string", "description": "Un URL personale all'immagine"}, "schema": {"readOnly": true, "type": "string", "description": "Un URL allo schema di immagini"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-07-03 00:02:28.175 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmp_pqhTv execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:29.378 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmp_pqhTv" returned: 0 in 1.203s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:29.387 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:30.858 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm" returned: 0 in 1.471s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:30.862 803 DEBUG cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] 4b995790-8d09-478f-9121-60efe5389473 was qcow2, converting to raw  fetch_to_volume_format /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:421
2018-07-03 00:02:30.864 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--9418bd73--30a7--4927--b450--e2af3cfb105d oflag=direct execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:32.429 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--9418bd73--30a7--4927--b450--e2af3cfb105d oflag=direct" returned: 0 in 1.565s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:32.434 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--9418bd73--30a7--4927--b450--e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:35.444 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--9418bd73--30a7--4927--b450--e2af3cfb105d" returned: 0 in 3.010s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:35.447 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:36.729 803 DEBUG oslo_concurrency.processutils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm" returned: 0 in 1.283s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:36.731 803 DEBUG cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Image conversion details: src /var/lib/cinder/conversion/tmplZwf1nlocalhost.localdomain@lvm, size 44.00 MB, duration 3.01 sec, destination /dev/mapper/cinder--volumes2-volume--9418bd73--30a7--4927--b450--e2af3cfb105d _convert_image /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:190
2018-07-03 00:02:36.732 803 INFO cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Converted 44.00 MB image at 14.60 MB/s
2018-07-03 00:02:36.733 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Downloaded image 4b995790-8d09-478f-9121-60efe5389473 ((None, None)) to volume 9418bd73-30a7-4927-b450-e2af3cfb105d successfully. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:582
2018-07-03 00:02:36.746 803 DEBUG cinder.image.image_utils [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Temporary image 4b995790-8d09-478f-9121-60efe5389473 for user cd8907a7c13344eb864b011f50896a6e is deleted. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:680
2018-07-03 00:02:36.747 803 DEBUG cinder.coordination [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "/var/lib/cinder/cinder-4b995790-8d09-478f-9121-60efe5389473" released by "_create_from_image_cache_or_download" :: held 13.305s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-07-03 00:02:36.749 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Marking volume 9418bd73-30a7-4927-b450-e2af3cfb105d as bootable. _enable_bootable_flag /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:479
2018-07-03 00:02:36.779 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Copia dei metadati da image 4b995790-8d09-478f-9121-60efe5389473 a 9418bd73-30a7-4927-b450-e2af3cfb105d. _handle_bootable_volume_glance_meta /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:437
2018-07-03 00:02:36.783 803 DEBUG cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Creating volume glance metadata for volume 9418bd73-30a7-4927-b450-e2af3cfb105d backed by image 4b995790-8d09-478f-9121-60efe5389473 with: {'container_format': u'bare', 'min_ram': 0, 'disk_format': u'qcow2', 'image_name': u'tempest-cirros-0.4.0-x86_64-1530568629', 'image_id': u'4b995790-8d09-478f-9121-60efe5389473', 'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', 'min_disk': 0, 'size': 12716032}. _capture_volume_image_metadata /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:616
2018-07-03 00:02:36.977 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (df8cb4e7-6686-410d-af03-a146e8e31dff) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:36.981 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (bac919c7-0531-4f33-8e18-abf126a4cdda) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-07-03 00:02:37.081 803 INFO cinder.volume.flows.manager.create_volume [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Volume volume-9418bd73-30a7-4927-b450-e2af3cfb105d (9418bd73-30a7-4927-b450-e2af3cfb105d): created successfully
2018-07-03 00:02:37.088 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (bac919c7-0531-4f33-8e18-abf126a4cdda) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-07-03 00:02:37.102 803 DEBUG cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Flow 'volume_create_manager' (e20493f8-d580-49d8-a5f4-217f5259d1f6) transitioned into state 'SUCCESS' from state 'RUNNING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-07-03 00:02:37.104 803 INFO cinder.volume.manager [req-cd56eb91-6afd-4a75-a0eb-53e83fba5800 cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Created volume successfully.
2018-07-03 00:02:39.566 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:02:39.568 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:02:39.631 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:02:39.636 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:02:39.637 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:40.947 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.310s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:40.950 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:42.067 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.118s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:42.075 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:43.282 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.207s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:02:43.286 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:02:44.374 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.089s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:16.928 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:18.016 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d" returned: 0 in 1.088s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:18.123 803 INFO cinder.volume.targets.lio [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Creating iscsi_target for volume: volume-9418bd73-30a7-4927-b450-e2af3cfb105d
2018-07-03 00:03:18.125 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:18.127 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d JYFr4S8j2VYHG2oouC4h Mz7oN22MVC875RBG False -p3260 -a10.0.20.23 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:20.239 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d JYFr4S8j2VYHG2oouC4h Mz7oN22MVC875RBG False -p3260 -a10.0.20.23" returned: 0 in 2.113s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:20.242 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.116s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:20.243 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:20.245 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:22.930 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.686s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:22.931 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.688s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:22.932 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:22.934 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:25.652 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.719s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:25.655 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.723s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:25.656 803 DEBUG cinder.volume.targets.iscsi [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Set provider_location to: 10.0.20.23:3260,iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d 0 create_export /usr/lib/python2.7/site-packages/cinder/volume/targets/iscsi.py:215
2018-07-03 00:03:25.686 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:25.690 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d JYFr4S8j2VYHG2oouC4h Mz7oN22MVC875RBG iqn.1994-05.com.redhat:15e63d5e5056 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:27.633 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d JYFr4S8j2VYHG2oouC4h Mz7oN22MVC875RBG iqn.1994-05.com.redhat:15e63d5e5056" returned: 0 in 1.946s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:27.635 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.949s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:27.637 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:27.638 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:29.793 803 DEBUG oslo_concurrency.processutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.155s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:29.795 803 DEBUG oslo_concurrency.lockutils [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.158s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:29.848 803 INFO cinder.volume.manager [req-27522b57-e94a-45a5-bcaa-d4592a2eeccd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Initialize volume connection completed successfully.
2018-07-03 00:03:34.563 803 DEBUG oslo_concurrency.lockutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:34.568 803 DEBUG oslo_concurrency.processutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d iqn.1994-05.com.redhat:15e63d5e5056 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:37.140 803 DEBUG oslo_concurrency.processutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d iqn.1994-05.com.redhat:15e63d5e5056" returned: 0 in 2.573s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:37.145 803 DEBUG oslo_concurrency.lockutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.582s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:37.147 803 DEBUG oslo_concurrency.lockutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:03:37.148 803 DEBUG oslo_concurrency.processutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:39.575 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:03:39.576 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:03:39.744 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:03:39.746 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:03:39.762 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:41.043 803 DEBUG oslo_concurrency.processutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 3.895s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:41.045 803 DEBUG oslo_concurrency.lockutils [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 3.899s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:03:41.046 803 INFO cinder.volume.manager [req-5f8b851b-6dbe-450d-9f5c-13e0116704fd cd8907a7c13344eb864b011f50896a6e fc87b8481ee14800a696810d32b79c6f - a1c7fd30bedc43b3a4986149401e2584 a1c7fd30bedc43b3a4986149401e2584] Terminate volume connection completed successfully.
2018-07-03 00:03:41.631 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.869s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:41.633 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:42.700 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.067s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:42.703 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:43.763 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.060s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:03:43.766 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:03:44.727 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.961s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:04:39.580 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:04:39.582 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:04:39.610 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:04:39.611 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:04:39.613 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:04:40.681 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.068s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:04:40.683 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:04:41.493 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.810s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:04:41.495 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:04:42.500 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.005s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:04:42.503 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:04:43.219 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.716s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:05:39.582 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:05:39.585 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:05:39.640 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:05:39.641 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:05:39.644 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:05:40.906 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.262s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:05:40.910 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:05:42.033 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.123s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:05:42.036 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:05:42.904 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.869s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:05:42.908 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:05:43.684 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.776s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:39.587 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:06:39.589 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:06:39.638 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:06:39.639 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:06:39.641 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:40.831 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.191s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:40.834 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:41.707 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.874s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:41.710 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:42.743 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.033s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:42.746 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:43.528 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.783s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:51.148 803 DEBUG cinder.coordination [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "/var/lib/cinder/cinder-9418bd73-30a7-4927-b450-e2af3cfb105d-delete_volume" acquired by "delete_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-07-03 00:06:51.344 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.009s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:06:51.346 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:53.789 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.443s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:53.791 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.448s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:06:53.793 803 INFO cinder.volume.targets.lio [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Removing iscsi_target: 9418bd73-30a7-4927-b450-e2af3cfb105d
2018-07-03 00:06:53.794 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:06:53.796 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:55.781 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-9418bd73-30a7-4927-b450-e2af3cfb105d" returned: 0 in 1.985s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:55.783 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.989s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:06:55.785 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-07-03 00:06:55.788 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:58.061 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.273s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:58.065 803 DEBUG oslo_concurrency.lockutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.280s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-07-03 00:06:58.068 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:06:59.193 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d" returned: 0 in 1.126s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:06:59.196 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:00.229 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d" returned: 0 in 1.033s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:00.234 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:01.680 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-9418bd73-30a7-4927-b450-e2af3cfb105d" returned: 0 in 1.446s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:01.686 803 INFO cinder.volume.drivers.lvm [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Successfully deleted volume: 9418bd73-30a7-4927-b450-e2af3cfb105d
2018-07-03 00:07:01.786 803 WARNING cinder.quota [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Deprecated: Quota predefinita per la risorsa: gigabytes_iscsi è impostato dall'indicatore quota predefinita: quota_gigabytes_iscsi, è ora obsoleta. Utilizzare la classe di quota predefinita per la quota predefinita.
2018-07-03 00:07:01.790 803 WARNING cinder.quota [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Deprecated: Quota predefinita per la risorsa: volumes_iscsi è impostato dall'indicatore quota predefinita: quota_volumes_iscsi, è ora obsoleta. Utilizzare la classe di quota predefinita per la quota predefinita.
2018-07-03 00:07:01.998 803 DEBUG cinder.quota [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Created reservations ['42d08320-416b-4924-ad62-2555092fa65e', '3b72afa8-0c8f-456e-a0ba-d836923ce7aa', '55a272bd-eade-472f-b0e0-fa8479c4f12a', '68ded86e-66b7-4640-bfa4-42c36a99bdc4'] reserve /usr/lib/python2.7/site-packages/cinder/quota.py:1029
2018-07-03 00:07:02.247 803 DEBUG cinder.volume.utils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Can not find volume 9418bd73-30a7-4927-b450-e2af3cfb105d at notify usage _usage_from_volume /usr/lib/python2.7/site-packages/cinder/volume/utils.py:99
2018-07-03 00:07:02.481 803 DEBUG cinder.volume.drivers.lvm [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:07:02.485 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:03.995 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.510s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:04.001 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:05.501 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.503s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:05.506 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:07.219 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.714s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:07.224 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:08.397 803 DEBUG oslo_concurrency.processutils [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.173s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:08.402 803 DEBUG cinder.manager [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:07:08.682 803 INFO cinder.volume.manager [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Deleted volume successfully.
2018-07-03 00:07:08.722 803 DEBUG cinder.coordination [req-aa8927a3-0228-40ad-bad2-468e90389859 4e72ad8ceb8f47a28db5a4b3551f9335 007e7f1ed2554995b505b934dd328646 - default default] Lock "/var/lib/cinder/cinder-9418bd73-30a7-4927-b450-e2af3cfb105d-delete_volume" released by "delete_volume" :: held 17.574s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-07-03 00:07:39.584 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:07:39.587 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:07:39.629 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:07:39.630 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:07:39.632 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:40.572 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.941s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:40.574 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:41.427 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.853s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:41.431 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:42.339 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.909s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:07:42.342 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-07-03 00:07:43.210 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.869s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-07-03 00:08:39.582 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:08:39.584 803 DEBUG cinder.manager [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-07-03 00:08:39.609 803 DEBUG oslo_service.periodic_task [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-07-03 00:08:39.610 803 DEBUG cinder.volume.drivers.lvm [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-07-03 00:08:39.611 803 DEBUG oslo_concurrency.processutils [req-40695008-324c-4dd9-a89d-722357614433 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
